WordPressKit/WordPressKit/WordPressComServiceRemote.m

```diff
@@ -68,6 +68,44 @@ - (void)createWPComAccountWithEmail:(NSString *)email
     [self.wordPressComRestApi POST:requestUrl parameters:params success:successBlock failure:failureBlock];
 }
 
+// https://developer.wordpress.com/docs/api/console/
+// API v1 POST /users/social/new
+/*
+ client_id
+ client_secret
+ service
+ signup_flow_name
+ access_token // optional, I don't think we'll need it
+ id_token
+ */
```

Comment:
Is this comment really helpful? Listing params without describing them doesn't really clarify anything, and one can get the information from the API console. I can see adding a comment about the `access_token` within the method when the params dictionary is created...